docs/php-basic-auth.md

```diff
@@ -0,0 +1,104 @@
+# PHP Basic Auth
+
+In many instances, the Require Login component of Altis is sufficient to block access to websites. However, sometimes, it's desirable to be able to test elements -- particularly on development environments -- as a logged-out user. In these cases, the Require Login module does not solve the problem and another method of blocking access is required. That's where PHP Basic Auth comes in.
```

Comment:
Wanted to note that Markdown doesn't convert the double dashes `--` to em dashes. I think they might make more sense as just commas